sphinx.util.compat module not found

The latest PR from @sanchestm failed the CI build due to [an issue building the docs](https://travis-ci.org/dib-lab/khmer/jobs/341123873#L2876). Updating my Python packages with `pip install --upgrade -r doc/requirements.txt` I can reproduce the error message on my local machine.

```text
$ make doc
*** NOTE: Found Cython, extension files will be transpiled if this is an install invocation.
*** NOTE: Found Cython, extension files will be transpiled if this is an install invocation.
./setup.py build_sphinx --fresh-env
*** NOTE: Found Cython, extension files will be transpiled if this is an install invocation.
running build_sphinx
Running Sphinx v1.7.0

Extension error:
Could not import extension sphinxcontrib.autoprogram (exception: No module named 'sphinx.util.compat')
make: *** [build/sphinx/html/index.html] Error 1
```

Updating to the latest version of `sphinxcontrib.autoprogram` (0.1.3) did not fix the issue.

--- a/doc/requirements.txt
+++ b/doc/requirements.txt
@@ -1,4 +1,4 @@
-sphinxcontrib-autoprogram==0.1.2
+sphinxcontrib-autoprogram>=0.1.4
 setuptools>=3.4.1
 guzzle_sphinx_theme==0.7.11
 Cython==0.25.2

